packages/static/kairos-overlay-files/files/system/oem/00_rootfs.yaml

```diff
@@ -140,8 +140,18 @@ stages:
       name: "Restore /etc/machine-id for systemd systems"
       commands:
         - cat /usr/local/etc/machine-id > /etc/machine-id
+    # If we didn't restore it already, ensure /etc/machine-id is generated
+    - if: '[ -e "/sbin/systemctl" ] || [ -e "/bin/systemctl" ] || [ -e "/usr/sbin/systemctl" ] || [ -e "/usr/bin/systemctl" ]'
+      name: "Ensure /etc/machine-id for systemd systems"
+      commands:
+        # https://www.freedesktop.org/software/systemd/man/latest/systemd-machine-id-commit.service.html
+        # https://www.freedesktop.org/software/systemd/man/latest/systemd-machine-id-setup.html
+        # Mask the commit service to avoid systemd messing with machine-id.
+        - ln -s /dev/null /etc/systemd/system/systemd-machine-id-commit.service
```

Comment:
Normally, since this service will kick in later, it should have no effect, so we shouldn't need to mask it but better safe than sorry?

Reply:
Also, we should run it on the rootfs, which is the earliest stage we got I think, so we mask it asap.

Reply:
Good points, pushed a commit